.github/workflows/semantic-pull-request.yml

```diff
@@ -0,0 +1,46 @@
+name: "Check PR title format"
+
+ on:
+   pull_request_target:
+     types:
+       - opened
+       - edited
+       - synchronize
+
+ jobs:
+   main:
+     name: Validate PR title
+     runs-on: ubuntu-latest
+     steps:
+       - uses: amannn/action-semantic-pull-request@v5
+         env:
+           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+         with:
+           # Configure which types are allowed (newline delimited).
+           # Default: https://github.com/commitizen/conventional-commit-types
+           # Our specific types/scopes (from https://github.com/dashpay/dash/blob/master/CONTRIBUTING.md#contributor-workflow)
+           types: |
+             feat
+             fix
+             docs
+             style
+             refactor
+             perf
+             test
+             build
+             ci
+             chore
+             revert
+             trivial
+             backport
+           # Configure which scopes are allowed (newline delimited).
+           scopes: |
+             consensus
+             qt
+             log
+             mining
+             net
+             rpc/rest/zmq
+             scripts
+             utils
+             wallet
```

Comment:
Before I do a similar PR on android, I would like to see scopes we would be using on our apps.  Most of these may not apply.

             consensus - handled by DashJ and DashSync
             qt - we don't use QT, perhaps we would rename this "ui"
             log - 
             mining - handled by DashJ and DashSync, if at all
             net - handled by DashJ and DashSync, if at all
             rpc/rest/zmq - N/A
             scripts  - handled by DashJ and DashSync, if at all
             utils
             wallet - this would be the main function of our app, as opposed to others like crowdnode, coinbase, etc...
             
On android, we are also using other scopes that relate to big features:
             dashpay
             crowdnode
             uphold
             coinbase
             explore-dash

Otherwise, we are not using scopes for fixes and features, though we can start doing so if we have a list.  If we later add a new feature called Coffee, then we would need to add a coffee scope.